### PR TITLE
feat: Badge preview mobile

### PR DESCRIPTION
**Pull request by FedeMadoery**
# Description

### CODE REVIEW

File: src/pagePartials/badge/preview/BadgeOwnedPreviewMobile.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            </Typography>
+
+            <Typography variant="body4">
+              {t('badge.viewBadge.checkHowElse')}
+              <LinkWithTranslation pathname="/badge/explorer">
+                {t('badge.viewBadge.seeAll').toUpperCase()}
+              </LinkWithTranslation>
+            </Typography>
+          </Box>
+        </Stack>
```
Comment (ramabit) [inline src/pagePartials/badge/preview/BadgeOwnedPreviewMobile.tsx:99]: is there a way to reuse code for this and avoid copy-pasting?

Comment (ramabit) [reply src/pagePartials/badge/preview/BadgeOwnedPreviewMobile.tsx:99]: Ideally if possible, I like more the approach of having only 1 component with the styling and rendering adjusting to the viewport, as to render different components. However, perhaps for this case I am wrong and it is very complex, in that case let me know

Comment (FedeMadoery) [reply src/pagePartials/badge/preview/BadgeOwnedPreviewMobile.tsx:99]: Yep, I agree with you, but It's not even ready... I need to see how to show the Challenge data, and after that, I was planning to put everything together as you said

Comment (FedeMadoery) [reply src/pagePartials/badge/preview/BadgeOwnedPreviewMobile.tsx]: Let me know what do you think now 🤞 , should be ready

**Review by Agupane — APPROVED** [review]

**Pull request merged**